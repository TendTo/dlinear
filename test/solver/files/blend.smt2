(set-logic QF_LRA)
(declare-const |1| Real)
(declare-const |2| Real)
(declare-const |3| Real)
(declare-const |4| Real)
(declare-const |5| Real)
(declare-const |6| Real)
(declare-const |7| Real)
(declare-const |8| Real)
(declare-const |9| Real)
(declare-const |10| Real)
(declare-const |11| Real)
(declare-const |12| Real)
(declare-const |13| Real)
(declare-const |14| Real)
(declare-const |15| Real)
(declare-const |16| Real)
(declare-const |17| Real)
(declare-const |18| Real)
(declare-const |19| Real)
(declare-const |20| Real)
(declare-const |21| Real)
(declare-const |22| Real)
(declare-const |23| Real)
(declare-const |24| Real)
(declare-const |25| Real)
(declare-const |26| Real)
(declare-const |27| Real)
(declare-const |28| Real)
(declare-const |29| Real)
(declare-const |30| Real)
(declare-const |31| Real)
(declare-const |32| Real)
(declare-const |33| Real)
(declare-const |34| Real)
(declare-const |35| Real)
(declare-const |36| Real)
(declare-const |37| Real)
(declare-const |38| Real)
(declare-const |39| Real)
(declare-const |40| Real)
(declare-const |41| Real)
(declare-const |42| Real)
(declare-const |43| Real)
(declare-const |44| Real)
(declare-const |45| Real)
(declare-const |46| Real)
(declare-const |47| Real)
(declare-const |48| Real)
(declare-const |49| Real)
(declare-const |50| Real)
(declare-const |51| Real)
(declare-const |52| Real)
(declare-const |53| Real)
(declare-const |54| Real)
(declare-const |55| Real)
(declare-const |56| Real)
(declare-const |57| Real)
(declare-const |58| Real)
(declare-const |59| Real)
(declare-const |60| Real)
(declare-const |61| Real)
(declare-const |62| Real)
(declare-const |63| Real)
(declare-const |64| Real)
(declare-const |65| Real)
(declare-const |66| Real)
(declare-const |67| Real)
(declare-const |68| Real)
(declare-const |69| Real)
(declare-const |70| Real)
(declare-const |71| Real)
(declare-const |72| Real)
(declare-const |73| Real)
(declare-const |74| Real)
(declare-const |75| Real)
(declare-const |76| Real)
(declare-const |77| Real)
(declare-const |78| Real)
(declare-const |79| Real)
(declare-const |80| Real)
(declare-const |81| Real)
(declare-const |82| Real)
(declare-const |83| Real)
(assert (<= 0 |1|))
(assert (<= 0 |2|))
(assert (<= 0 |3|))
(assert (<= 0 |4|))
(assert (<= 0 |5|))
(assert (<= 0 |6|))
(assert (<= 0 |7|))
(assert (<= 0 |8|))
(assert (<= 0 |9|))
(assert (<= 0 |10|))
(assert (<= 0 |11|))
(assert (<= 0 |12|))
(assert (<= 0 |13|))
(assert (<= 0 |14|))
(assert (<= 0 |15|))
(assert (<= 0 |16|))
(assert (<= 0 |17|))
(assert (<= 0 |18|))
(assert (<= 0 |19|))
(assert (<= 0 |20|))
(assert (<= 0 |21|))
(assert (<= 0 |22|))
(assert (<= 0 |23|))
(assert (<= 0 |24|))
(assert (<= 0 |25|))
(assert (<= 0 |26|))
(assert (<= 0 |27|))
(assert (<= 0 |28|))
(assert (<= 0 |29|))
(assert (<= 0 |30|))
(assert (<= 0 |31|))
(assert (<= 0 |32|))
(assert (<= 0 |33|))
(assert (<= 0 |34|))
(assert (<= 0 |35|))
(assert (<= 0 |36|))
(assert (<= 0 |37|))
(assert (<= 0 |38|))
(assert (<= 0 |39|))
(assert (<= 0 |40|))
(assert (<= 0 |41|))
(assert (<= 0 |42|))
(assert (<= 0 |43|))
(assert (<= 0 |44|))
(assert (<= 0 |45|))
(assert (<= 0 |46|))
(assert (<= 0 |47|))
(assert (<= 0 |48|))
(assert (<= 0 |49|))
(assert (<= 0 |50|))
(assert (<= 0 |51|))
(assert (<= 0 |52|))
(assert (<= 0 |53|))
(assert (<= 0 |54|))
(assert (<= 0 |55|))
(assert (<= 0 |56|))
(assert (<= 0 |57|))
(assert (<= 0 |58|))
(assert (<= 0 |59|))
(assert (<= 0 |60|))
(assert (<= 0 |61|))
(assert (<= 0 |62|))
(assert (<= 0 |63|))
(assert (<= 0 |64|))
(assert (<= 0 |65|))
(assert (<= 0 |66|))
(assert (<= 0 |67|))
(assert (<= 0 |68|))
(assert (<= 0 |69|))
(assert (<= 0 |70|))
(assert (<= 0 |71|))
(assert (<= 0 |72|))
(assert (<= 0 |73|))
(assert (<= 0 |74|))
(assert (<= 0 |75|))
(assert (<= 0 |76|))
(assert (<= 0 |77|))
(assert (<= 0 |78|))
(assert (<= 0 |79|))
(assert (<= 0 |80|))
(assert (<= 0 |81|))
(assert (<= 0 |82|))
(assert (<= 0 |83|))
(assert (= (+ (* (- 0.2931) |2|) |4|) 0))
(assert (= (+ (* (- 0.537) |1|) |3|) 0))
(assert (= (+ (* (- 0.131) |1|) (* (- 0.117) |2|) |22| |62|) 0))
(assert (= (+ (* (- 0.1155) |1|) (* (- 0.0649) |2|) |17| |21| |61|) 0))
(assert (= (+ (* (- 0.0365) |1|) (* (- 0.1233) |2|) |15| |19| |67|) 0))
(assert (= (+ (* (- 0.143) |1|) (* (- 0.2217) |2|) |16| |20| |66|) 0))
(assert (= (+ (* (- 0.037) |1|) |64|) 0))
(assert (= (+ (* (- 0.18) |2|) |65|) 0))
(assert (= (+ (* (- 0.0277) |3|) (* (- 0.0112) |4|) (* (- 0.175) |5|) (* (- 0.271) |6|) (* (- 0.2836) |7|) (* (- 0.0571) |24|) (* (- 1) |25|) (* (- 1) |54|) |55| |74|) 0))
(assert (= (+ (* (- 0.0563) |3|) (* (- 0.0378) |4|) (* (- 0.27) |5|) (* (- 0.3285) |6|) (* (- 0.3285) |7|) (* (- 0.0185) |19|) (* (- 0.0185) |20|) (* (- 0.0184) |21|) (* (- 0.0184) |22|) (* (- 0.0114) |24|) (* (- 1) |26|) (* (- 1) |27|) |35| |43| |51| |56| |73| (* (- 0.5) |79|)) 0))
(assert (= (+ (* (- 0.199) |3|) (* (- 0.1502) |4|) (* (- 0.028) |5|) (* (- 0.0255) |6|) (* (- 0.0241) |7|) |28| |38| |46| |58|) 0))
(assert (= (+ (* (- 0.6873) |3|) (* (- 0.7953) |4|) |8| |59|) 0))
(assert (= (+ (* (- 0.017) |3|) (* (- 0.0099) |4|) (* (- 0.455) |5|) (* (- 0.2656) |6|) (* (- 0.2502) |7|) (* (- 0.0568) |19|) (* (- 0.0568) |20|) (* (- 0.0564) |21|) (* (- 0.0564) |22|) (* 0.76 |23|) (* 0.6571 |24|) |27| |72| (* (- 0.5) |79|)) 0))
(assert (= (+ (* (- 1) |8|) |11| |12|) 0))
(assert (= (+ (* (- 1) |9|) (* (- 1) |11|) |13|) 0))
(assert (= (+ (* (- 1) |10|) (* (- 1) |12|) |14|) 0))
(assert (= (+ |7| (* (- 0.0588) |13|)) 0))
(assert (= (+ |6| (* (- 0.0404) |14|)) 0))
(assert (= (+ (* (- 0.8145) |13|) |30| |40| |48|) 0))
(assert (= (+ (* (- 0.8564) |14|) |31| |41| |49|) 0))
(assert (= (+ |5| (* (- 0.3321) |15|) (* (- 0.3321) |16|) (* (- 0.2414) |17|) (* (- 0.2414) |18|)) 0))
(assert (= (+ |9| |10| (* (- 0.5875) |15|) (* (- 0.5875) |16|) (* (- 0.6627) |17|) (* (- 0.6627) |18|)) 0))
(assert (= (+ (* (- 0.0091) |13|) (* (- 0.0069) |14|) (* (- 0.362) |15|) (* (- 0.362) |16|) (* (- 0.293) |17|) (* (- 0.293) |18|) |29| |39| |47|) 0))
(assert (= (+ (* (- 0.0806) |19|) (* (- 0.0806) |20|) (* (- 0.078) |21|) (* (- 0.078) |22|) (* 0.5714 |24|) |26| |76|) 0))
(assert (= (+ (* (- 0.0658) |19|) (* (- 0.0658) |20|) (* (- 0.0655) |21|) (* (- 0.0655) |22|) (* 0.5714 |23|) |25| |75|) 0))
(assert (= (+ (* (- 0.0328) |19|) (* (- 0.0328) |20|) (* (- 0.0303) |21|) (* (- 0.0303) |22|) |54|) 0))
(assert (= (+ (* (- 0.4934) |19|) (* (- 0.4934) |20|) (* (- 0.475) |21|) (* (- 0.475) |22|) |32| |42| |50|) 0))
(assert (= (+ |18| (* (- 0.2922) |19|) (* (- 0.2922) |20|) (* (- 0.305) |21|) (* (- 0.305) |22|) |69|) 0))
(assert (= (+ (* (- 0.0096) |19|) (* (- 0.0096) |20|) |68|) 0))
(assert (= (+ (* (- 1) |23|) |33|) 0))
(assert (= (+ (* (- 1) |24|) |34|) 0))
(assert (= (+ (* (- 1) |28|) (* (- 1) |29|) (* (- 1) |30|) (* (- 1) |31|) (* (- 1) |32|) (* (- 1) |33|) (* (- 1) |34|) (* (- 1) |35|) |37|) 0))
(assert (= (+ (* (- 1) |38|) (* (- 1) |39|) (* (- 1) |40|) (* (- 1) |41|) (* (- 1) |42|) (* (- 1) |43|) |45|) 0))
(assert (= (+ (* (- 1) |61|) (* (- 1) |62|) |63|) 0))
(assert (= (+ (* (- 1) |64|) (* (- 1) |65|) (* (- 1) |66|) (* (- 1) |67|) (* (- 1) |68|) (* (- 1) |69|) |70|) 0))
(assert (= (+ (* (- 1) |46|) (* (- 1) |47|) (* (- 1) |48|) (* (- 1) |49|) (* (- 1) |50|) (* (- 1) |51|) |53|) 0))
(assert (= (+ (* (- 1) |55|) (* (- 1) |56|) |57|) 0))
(assert (= (+ (* (- 1) |58|) (* (- 1) |59|) |60|) 0))
(assert (= (+ (* 0.0042 |2|) (* 0.0327 |8|) (* (- 0.8239) |13|) (* (- 0.7689) |14|) (* 2.3 |15|) (* 2.3 |16|) (* 2.3 |17|) (* 2.3 |18|) |71|) 0))
(assert (= (+ (* 0.003 |1|) (* 0.003 |2|) (* 0.01303 |3|) (* 0.01303 |4|) (* 0.01303 |5|) (* 0.01303 |6|) (* 0.01303 |7|) (* 0.0081 |13|) (* 0.0063 |14|) (* (- 0.0654) |19|) (* (- 0.0654) |20|) (* (- 0.0654) |21|) (* (- 0.0654) |22|) (* 0.1869 |23|) (* 0.1724 |24|) (* (- 1) |77|) |78|) 0))
(assert (= (+ (* 0.0587 |1|) (* 0.1053 |2|) (* 0.0506 |3|) (* 0.0448 |4|) (* 0.0506 |5|) (* 0.0506 |6|) (* 0.0506 |7|) (* 0.094 |8|) (* (- 0.2112) |13|) (* (- 0.156) |14|) (* (- 0.2049) |15|) (* (- 0.2049) |16|) (* (- 0.1531) |17|) (* (- 0.1531) |18|) (* (- 0.2535) |19|) (* (- 0.2535) |20|) (* (- 0.2703) |21|) (* (- 0.2703) |22|) (* 0.2796 |23|) (* 0.2579 |24|) (* (- 0.325) |71|) (* (- 4.153) |72|) (* (- 4.316) |73|) (* (- 3.814) |74|) (* (- 3.808) |75|) (* (- 4.44) |76|) (* 1.42 |77|) (* (- 1) |80|) |81|) 0))
(assert (= (+ (* 0.15 |1|) (* 0.185 |2|) (* 0.209 |3|) (* 0.185 |4|) (* 0.209 |5|) (* 0.209 |6|) (* 0.209 |7|) (* 0.045 |8|) (* 0.387 |13|) (* 0.297 |14|) (* 0.826 |15|) (* 0.826 |16|) (* 0.826 |17|) (* 0.826 |18|) (* 0.632 |19|) (* 0.632 |20|) (* 0.632 |21|) (* 0.632 |22|) (* 2.241 |23|) (* 2.067 |24|) (* (- 1) |82|)) 0))
(assert (= (+ (* 0.302 |1|) (* 0.384 |2|) (* 0.495 |3|) (* 0.721 |4|) (* 0.495 |5|) (* 0.495 |6|) (* 0.495 |7|) (* 0.793 |8|) (* 1.03 |13|) (* 0.792 |14|) (* 14.61 |15|) (* 14.61 |16|) (* 14.61 |17|) (* 14.61 |18|) (* 0.6807 |19|) (* 0.6807 |20|) (* 0.6807 |21|) (* 0.6807 |22|) (* 2.766 |23|) (* 2.552 |24|) (* (- 1) |83|)) 0))
(assert (<= (+ (* (- 7.95) |28|) (* (- 8.84) |29|) (* (- 9.43) |30|) (* (- 9.03) |31|) (* (- 9.23) |32|) (* (- 9.4) |33|) (* (- 9.74) |34|) (* (- 9.74) |35|) (* (- 0.493) |36|) (* 10.03 |37|)) 0))
(assert (<= (+ (* (- 8.7) |28|) (* (- 9.45) |29|) (* (- 9.57) |30|) (* (- 9.32) |31|) (* (- 9.22) |32|) (* (- 9.85) |33|) (* (- 10.1) |34|) (* (- 9.9) |35|) (* (- 0.165) |36|) (* 10.03 |37|)) 0))
(assert (<= (+ (* (- 3) |28|) (* (- 3) |29|) (* (- 3) |30|) (* (- 3) |31|) (* (- 3) |32|) (* (- 3) |33|) (* (- 3) |34|) (* (- 3) |35|) |36|) 0))
(assert (<= (+ (* 14 |28|) (* 12 |29|) (* 3.5 |30|) (* 3.5 |31|) (* 6 |32|) (* 2.5 |33|) (* 3.3 |34|) (* 66 |35|) (* (- 9.5) |37|)) 0))
(assert (<= (+ |28| |29| (* 0.233 |30|) (* 0.205 |31|) (* 0.381 |32|) (* 0.39 |33|) (* 0.233 |34|) |35| (* (- 0.5) |37|)) 0))
(assert (<= (+ (* (- 1) |28|) (* (- 1) |29|) (* (- 0.358) |30|) (* (- 0.333) |31|) (* (- 0.509) |32|) (* (- 0.77) |33|) (* (- 0.58) |34|) (* (- 1) |35|) (* 0.5 |37|)) 0))
(assert (<= (+ (* (- 0.00862) |2|) (* (- 7.98) |38|) (* (- 8.87) |39|) (* (- 9.46) |40|) (* (- 9.06) |41|) (* (- 9.26) |42|) (* (- 9.77) |43|) (* (- 0.435) |44|) (* 9.65 |45|)) 0))
(assert (<= (+ (* (- 0.00862) |2|) (* (- 8.58) |38|) (* (- 9.33) |39|) (* (- 9.45) |40|) (* (- 9.2) |41|) (* (- 9.13) |42|) (* (- 9.78) |43|) (* (- 0.208) |44|) (* 9.65 |45|)) 0))
(assert (<= (+ (* (- 3) |38|) (* (- 3) |39|) (* (- 3) |40|) (* (- 3) |41|) (* (- 3) |42|) (* (- 3) |43|) |44|) 0))
(assert (<= (+ (* 14 |38|) (* 12 |39|) (* 3.5 |40|) (* 3.5 |41|) (* 6 |42|) (* 66 |43|) (* (- 9.5) |45|)) 0))
(assert (<= (+ |38| |39| (* 0.233 |40|) (* 0.205 |41|) (* 0.318 |42|) |43| (* (- 0.5) |45|)) 0))
(assert (<= (+ (* (- 1) |38|) (* (- 1) |39|) (* (- 0.358) |40|) (* (- 0.333) |41|) (* (- 0.509) |42|) (* (- 1) |43|) (* 0.5 |45|)) 0))
(assert (<= (+ (* (- 0.0101) |2|) (* (- 7.99) |46|) (* (- 8.88) |47|) (* (- 9.47) |48|) (* (- 9.07) |49|) (* (- 9.27) |50|) (* (- 9.78) |51|) (* (- 0.426) |52|) (* 9.05 |53|)) 0))
(assert (<= (+ (* (- 0.0101) |2|) (* (- 8.59) |46|) (* (- 9.34) |47|) (* (- 9.46) |48|) (* (- 9.21) |49|) (* (- 9.14) |50|) (* (- 9.79) |51|) (* (- 0.204) |52|) (* 9.05 |53|)) 0))
(assert (<= (+ (* (- 3) |46|) (* (- 3) |47|) (* (- 3) |48|) (* (- 3) |49|) (* (- 3) |50|) (* (- 3) |51|) |52|) 0))
(assert (<= (+ (* 14 |46|) (* 12 |47|) (* 3.5 |48|) (* 3.5 |49|) (* 6 |50|) (* 66 |51|) (* (- 9.5) |53|)) 0))
(assert (<= (+ |46| |47| (* 0.233 |48|) (* 0.205 |49|) (* 0.318 |50|) |51| (* (- 0.5) |53|)) 0))
(assert (<= (+ (* (- 1) |46|) (* (- 1) |47|) (* (- 0.358) |48|) (* (- 0.333) |49|) (* (- 0.509) |50|) (* (- 1) |51|) (* 0.5 |53|)) 0))
(assert (<= (+ (* 10.1 |64|) (* 12.63 |65|) (* 8.05 |66|) (* 6.9 |67|) (* 8.05 |68|) (* 4.4 |69|) (* (- 10.1) |70|)) 0))
(assert (<= (+ (* (- 14) |58|) (* (- 0.8) |59|) (* 2 |60|)) 0))
(assert (<= (+ (* 14 |58|) (* 0.8 |59|) (* (- 3) |60|)) 0))
(assert (<= (+ |3| |4| |15| |17| |19| |21| |22| |63| |67|) 23.26))
(assert (<= (+ |16| |20| |66|) 5.25))
(assert (<= |1| 26.32))
(assert (<= |2| 21.05))
(assert (<= (+ (* 1.3 |13|) |14|) 13.45))
(assert (<= (+ |15| |16| |17| |18|) 2.58))
(assert (<= (+ |19| |20| |21| |22|) 10))
(assert (<= (+ |23| |24|) 10))
(assert (<= (+ (* 0.64 |37|) (* (- 0.36) |45|) (* (- 0.36) |53|)) 0))
(assert (<= (+ (* 0.35 |37|) (* 0.35 |45|) (* (- 0.65) |53|)) 0))
(check-sat)
