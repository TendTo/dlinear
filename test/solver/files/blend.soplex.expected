delta-sat with delta = 0.0009999999999999998 ( > 2305843009213693/2305843009213693952)
